rtt_rosparam/src/rtt_rosparam_service.cpp

```diff
@@ -239,7 +247,27 @@ class ROSParamService: public RTT::Service
   bool setParamComponentPrivate(const std::string &name) { return set(name, COMPONENT_PRIVATE); }
   bool setParamComponentRelative(const std::string &name) { return set(name, COMPONENT_RELATIVE); }
   bool setParamComponentAbsolute(const std::string &name) { return set(name, COMPONENT_ABSOLUTE); }
-};
+
+  /**
+   * Adds a ROS parameter of supported types by ROS param as a property to this
+   * bag.
+   * A Property is created which causes contents of the
+   * property always to be in sync
+   * with the contents of the ROS parameter.
+   * @param name The name of this property
```

Comment:
... and also the name of the linked ROS parameter, right?

```suggestion
   * @param name The name of the ROS parameter and property to be added
```